### PR TITLE
added Super del Norte

### LINKED ISSUE #3508: Add Super del Norte
http://www.superdelnorte.com.mx/

Needs Wikidata and Wikipedia

### PR DESCRIPTION
**Pull request by ferdinand0101**
fixes #3508

### CODE REVIEW

**Comment by ferdinand0101** [comment]
fixes #3508

**Comment by Adamant36** [comment]
Hi. The Wikidata entry is wrong. It's actually Q88388513.

File: brands/shop/supermarket.json [hunk 1]
```diff
@@ -4001,6 +4001,15 @@
       "name": "Superspar",
       "shop": "supermarket"
     }
+  },
+    "shop/supermarket|Super del Norte": {
+    "countryCodes": ["mx"],
+    "tags": {
+      "brand": "Super del Norte",
+      "brand:wikidata": "Q88388513Q88388513",
```
Comment (bmillemathias) [inline brands/shop/supermarket.json]: Could you fix the wikidata to have all in one commit as suggested by @Adamant36 ?

Best.

Comment (ferdinand0101) [reply brands/shop/supermarket.json]: Idon't have the permissions to do that

**Pull request merged**